src/DynamoApplications/StartupUtils.cs

```diff
@@ -273,6 +273,7 @@ public static DynamoModel MakeModel(bool CLImode, string asmPath = "", string ho
         /// <returns></returns>
         public static DynamoModel MakeModel(bool CLImode, string asmPath = "", HostAnalyticsInfo info = new HostAnalyticsInfo())
         {
+            new DynamoModel();
```

Comment:
what is this for?

Reply:
> what is this for?

I have created this constructor in order to access to the preferences in a direct way due its scope and apply the proper language, otherwise it would require more changes in the API.

Reply:
I don't think we can do this. This is basically a hack to read preferences setting right?

Reply:
yes / I will try to dissect the preference from the Model